settings.gradle.kts

```diff
@@ -23,6 +23,13 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
+pluginManagement {
+    repositories {
+        maven("https://dl.bintray.com/kotlin/kotlin-eap")
```

Comment:
Bintray is gonna be [shut down](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/) in a month. Also, I haven't noticed any requirement in the EAP [setup](https://kotlinlang.org/docs/configure-build-for-eap.html#configure-in-gradle) for a bintray repo. Maybe we can remove this repository?